### PR TITLE
Add README section describing the licensing situation

### PR DESCRIPTION
**Pull request by jcbrand**
CC: @nick-denry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,21 +27,17 @@ It should happen once. A very basic integration of [Converse.js](https://convers
...
+#### License information
 
-Like Converse.js, Converse-Desktop's files are released under the Mozilla
+Like Converse.js, Converse Desktop's files are released under the Mozilla
 Public License version 2 (MPLv2).
 
-The gist of this license is that the covered files must stay open source, and
-modifications to them need to be released under the same license, but new files
-(for example for your own plugin) don't have to be released under the same
-license.
```
Comment (jcbrand) [inline README.md]: @nick-denry Why did you remove this part?

Comment (nick-denry) [reply README.md]: There are few reasons:

1. It's look like too obvious and as our own explanation of MPLv2 and GPLv3 at this part:

>The gist of this license is that the covered files must stay open source, and
modifications to them need to be released under the same license

2. AFAIR this is not like this in case of entire project is under GPLv3?

>but new files (for example for your own plugin) don't have to be released under the same
> license.

So, I think we should let the end-user decide what license must be used in every particular case. If no, let's add this removed part again.

Comment (jcbrand) [reply README.md]: > 1. It's look like too obvious

I don't think it's obvious. Not all open source licenses require you to release licensed files under the same license. For example the MIT license doesn't (but you need to keep the copyright header).

> AFAIR this is not like this in case of entire project is under GPLv3?

Yes, but we're describing the MPLv2 license in that paragraph, not the GPLv3.

Comment (nick-denry) [reply README.md]: Aha, now I see. Thanks for explanation.
